recipes/prettier/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "prettier" %}
+{% set version = "2.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/prettier/prettier/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 31c22f314066d3997aeafb345ebc1143179500d62f2334efc965ccdc2b4ffcf7
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: npm install -g $(npm pack)
```

Comment:
Please have a look at https://github.com/conda-forge/vsce-feedstock/blob/master/recipe/build.sh for a nodejs package build. Otherwise licenses are not correctly included.